example/hyprland.conf

```diff
@@ -18,7 +18,7 @@ input {
 
 general {
     max_fps=60 # deprecated, unused
```

Comment:
@vaxerski shouldn't we just remove this if it's unused?

Reply:
yeah, seems kind of unnecessary